### DISCUSSION TITLE
Packet count is not updated on swanctl -l output for windows

### DISCUSSION THREAD

**Discussion by salmanmalik-emb** [strongswan/strongswan#1794] [General]
@tobiasbrunner  We have strongswan on both client (Windows) and server (Ubuntu). The connection is established successfully and pings are working from both sides.

The output of swanctl -l on windows side shows 0 packet count though, is this known or are we missing anything here?

### Swanctl -l output on windows 
> $ ./swanctl.exe -l
conn-1: #1, ESTABLISHED, IKEv2, 7b2face004a72b4a_i* 740efe0088189590_r
  local  '10.10.10.12' @ 192.168.2.88[4500]
  remote '10.10.10.10' @ 18.142.228.69[4500]
  AES_CBC-256/HMAC_SHA2_256_128/PRF_HMAC_SHA2_256/MODP_2048
  established 2317s ago, rekeying in 12042s
  conn-1: #2, reqid 1, INSTALLED, TUNNEL-in-UDP, ESP:AES_CBC-256/HMAC_SHA2_256_128
    installed 2313s ago, rekeying in 1085s, expires in 1648s
    in  ce793f46,      0 bytes,     0 packets
    out ca32892e,      0 bytes,     0 packets
    local  10.10.10.12/32
    remote 10.10.10.10/32

### Swanctl -l output on Ubuntu
> root@ip-172-31-24-60:/home/ubuntu# swanctl -l
conn-2: #34, ESTABLISHED, IKEv2, 7b2face004a72b4a_i 740efe0088189590_r*
  local  '10.10.10.10' @ 172.31.24.60[4500]
  remote '10.10.10.12' @ 182.191.83.208[4500]
  AES_CBC-256/HMAC_SHA2_256_128/PRF_HMAC_SHA2_256/MODP_2048
  established 2347s ago, rekeying in 11453s
  conn-2: #12, reqid 2, INSTALLED, TUNNEL-in-UDP, ESP:AES_CBC-256/HMAC_SHA2_256_128
    installed 2342s ago, rekeying in 1010s, expires in 1618s
    in  ca32892e,    432 bytes,     6 packets,  2271s ago
    out ce793f46,    432 bytes,     6 packets,  2270s ago
    local  10.10.10.10/32
    remote 10.10.10.12/32

Is there any plugin we are missing for this? Currently following plugins are loaded for windows
` loaded plugins: charon-svc nonce x509 pubkey pkcs1 pem openssl kernel-wfp kernel-iph socket-win vici`

**Comment by tobiasbrunner** [comment]
[As documented](https://docs.strongswan.org/docs/5.9/plugins/kernel-wfp.html#_sa_usage_lifetime_and_volumes), the _kernel-wfp_ plugin is currently unable to provide traffic stats.
